macros/src/types/named.rs

```diff
@@ -57,6 +63,7 @@ pub(crate) fn named(
 // build an expresion which expands to a string, representing a single field of a struct.
 fn format_field(
```

Comment:
Same here. Maybe just 1 sentence explaining when the expression ends up in flattened_fields and when in formatted_fields, and why these two vecs are required.

Reply:
Done!